docs/proposals/20200804-windows-support.md

```diff
@@ -0,0 +1,271 @@
+    - [Implementation Details/Notes/Constraints](#implementation-detailsnotesconstraints)
+      - [Signing of the components.](#signing-of-the-components)
+      - [Known prototypes and prior work:](#known-prototypes-and-prior-work)
+    - [Security Model](#security-model)
+    - [Risks and Mitigations](#risks-and-mitigations)
+  - [Alternatives](#alternatives)
+  - [Upgrade Strategy](#upgrade-strategy)
+  - [Additional Details](#additional-details)
+    - [Test Plan [optional]](#test-plan-optional)
+    - [Graduation Criteria [optional]](#graduation-criteria-optional)
+      - [Alpha](#alpha)
+      - [Beta](#beta)
+      - [Stable](#stable)
+    - [Version Skew Strategy](#version-skew-strategy)
+  - [Implementation History](#implementation-history)
+
+## Glossary
+
+Refer to the [Cluster API Book Glossary](https://cluster-api.sigs.k8s.io/reference/glossary.html).
+
+## Summary
+
+This proposal is for the support of the Windows nodes with CAPI and [infrastructure providers](https://cluster-api.sigs.k8s.io/reference/glossary.html#infrastructure-provider) that wish to support Windows [OS](https://cluster-api.sigs.k8s.io/reference/glossary.html#operating-system) worker nodes.  
+Windows has been Stable in Kubernetes since 1.14 and is supported in mix worker node scenarios.  
+
+Windows support will provide the ability to add Windows nodes to a [workload cluster](https://cluster-api.sigs.k8s.io/reference/glossary.html#workload-cluster).  Windows nodes are configured differently across 
+solutions and the centralization of the scripts and guidance in the Cluster API solution will provide 
+guidance for the community on best practices for configuring Windows nodes at scale.
+
+Windows node support has some unique challenges because of the current limitations of Windows Containers. 
+Windows containers do not support privileged operations which means that configuration and access to the host
+machine must be done at provisioning time.  
+
+An example of this limitation is how kube-proxy gets configured on Windows nodes. Kube-proxy typically run as a 
+Windows service on the host machine and it cannot be deployed as a Daemon set as it is on Linux.  
+To address this limitation the community has built tools such as the [CSI-Proxy](https://github.com/kubernetes-csi/csi-proxy), which is a CSI driver specific proxy. This proposal will address how to approach the configuration of components 
+that are typically deployed as daemon sets when bootstrapping Windows nodes in CAPI.
+
+## Motivation
+Kubernetes has supported Windows workloads since the release of Windows support in Kubernetes 1.14. The motivation of this project is to enable Cluster API customers to deploy Windows as part of a mixed OS cluster via the Cluster API automation built for platform operators.  This will enable cluster operators to define Windows machines in the same consistent and repeatable fashion.
```

Comment:
```suggestion
Kubernetes has supported Windows workloads since the release of Windows support in Kubernetes 1.14. The motivation of this proposal is to enable Cluster API users to deploy Windows as part of a mixed OS cluster via the Cluster API automation built for platform operators.  This will enable cluster operators to define Windows machines in the same consistent and repeatable fashion.
```